server/gc/src/main/java/org/apache/accumulo/gc/GCRun.java

```diff
@@ -201,13 +202,16 @@ public Stream<Reference> getReferences() {
 
       // combine all the entries read from file and scan columns in the metadata table
       Stream<StoredTabletFile> fileStream = tm.getFiles().stream();
+      // map the files to Reference objects
+      var stream = fileStream.map(f -> new ReferenceFile(tm.getTableId(), f.getMetaUpdateDelete()));
```

Comment:
Not sure if it matters, but could just get the tableId once for this loop.

Reply:
Also, renaming this variable to something like "fileRefStream" (or "fileStream", but renaming the current "fileStream"), will make the concatenation further down more clear. The generic name "stream" can get confusing if you're not deeply familiar with the code, especially since this variable is being reused as the concatenated stream as well.

Reply:
tableId lookup has been reduced to a single var. Stream names have been refactored to make their usage clearer.